packages/util/tests/integration/helpers/ensure-safe-component-test.js

```diff
@@ -175,4 +176,34 @@ module('Integration | Helper | ensure-safe-component', function (hooks) {
     assert.strictEqual(this.element.textContent.trim(), 'second:goodbye');
     assert.deepEqual(log, ['target instantiated', 'new target instantiated']);
   });
+
+  module('with strict: true', function () {
+    test('reference to class property-as-component is allowed', async function (assert) {
```

Comment:
passes on `release`
Fails on ember-source@4.4 and @3.28

Reply:
now that I'm using the correct flag, `strictMode`, we get:
4.4 + 3.28 (still):
```
Error: Attempted to load a helper, but there wasn't a helper manager associated with the definition. The definition was: ensureSafeComponent
```                
release:
```
Unsafe dynamic component: cannot statically analyze this expression in tests/integration/helpers/ensure-safe-component-
```